library/lib/build.js

```diff
@@ -106,14 +109,21 @@ module.exports = function build({ watch }) {
         publicPath,
         libraryTarget: 'commonjs2'
       },
-      externals: nodeExternals({ whitelist: ['@kaliber/config', kaliberBuildClientModules, /\.css$/] }),
+      externals: [
+        nodeExternals(externalConfForModulesDir('node_modules')),
+        yarnWorkspaceDir && nodeExternals(externalConfForModulesDir(path.resolve(yarnWorkspaceDir, 'node_modules')))
+      ].filter(Boolean),
       resolve: {
         extensions: ['.js'],
         modules: ['node_modules'],
         plugins: [absolutePathResolverPlugin(srcDir), fragmentResolverPlugin()]
       },
       resolveLoader: {
-        modules: [path.resolve(__dirname, '../webpack-loaders'), 'node_modules']
+        modules: [
+          path.resolve(__dirname, '../webpack-loaders'),
+          'node_modules',
+          yarnWorkspaceDir && path.resolve(yarnWorkspaceDir, 'node_modules')
```

Comment:
I wonder if this is required. My guess would be that `node_modules` would recursively searched. I might be mistaken though, and finding this in the documentation or source of webpack is a nightmare.

So if you added this to resolve a problem it's fine.

Reply:
Hmm, I added it as a precaution, haven't tested it without it.

Reply:
on a sidenote: where is the webpack 3.x documentation ?!

Reply:
Good point, haha, seems they only display the latest version...

This contains (hopefully most of) the information required to upgrade: https://github.com/webpack/webpack/releases

We should probably move to webpack 4 pretty soon